examples/bond_linuxbridge_vlan_absent.yml

```diff
@@ -0,0 +1,20 @@
+---
+interfaces:
+  - name: eth1
+    type: ethernet
+    state: absent
+  - name: eth2
+    type: ethernet
+    state: absent
+  - name: bond0
+    type: bond
+    state: absent
+  - name: bond0-br0
```

Comment:
It is common to start with the device itself and not with its slave.
I would even argue that embedding the port in the name is not appropriate with bridges and bonds.

Reply:
The order makes it easier to follow the network diagram in the up state.
Not sure what you mean with `port in the name`. The bridge names correspond to the VLAN that they belong to (none, or VLAN 29). I do not see why this would not be appropriate. It makes it easier to comprehend IMHO.

Reply:
It is not easy to comprehend and is not common in places I saw similar setups.
Slaves tend to change so a master name cannot follow that.

Reply:
Please propose an alternative naming.
This is about an example derived from a production example and for me it is easy to comprehend (I did not invent it myself). Also as an static example, it does not need to care about possible changes to the slaves because then it would be another example than showing how to create a bridge for vlan 29 on top of a bond for eth1 and eth2.

Reply:
I edited my last response with my suggestion. ‘br0-bond0’ I can understand.

Reply:
I changed the bridges to `br0` and `brXY`. To me a top-bottom notation like `br0-bond0` is confusing. Hope this is a working compromise.